### PR TITLE
Fix: hugo serve does not serve Font Awesome anymore #1281

### LINKED ISSUE #1281: hugo serve does not serve font awesome anymore
related to #1165 

It does not work for me:

```
GET http://127.0.0.1:1313/webfonts/fa-solid-900.woff net::ERR_ABORTED 404 (Not Found)
main.css:1 
```

off_000

```
./hugo mod graph
github.com/o11ydev/prometheus-workshop github.com/google/docsy@v0.5.0
github.com/o11ydev/prometheus-workshop github.com/google/docsy/dependencies@v0.5.0
github.com/google/docsy/dependencies@v0.5.0 github.com/twbs/bootstrap@v4.6.1+incompatible
github.com/google/docsy/dependencies@v0.5.0 github.com/FortAwesome/Font-Awesome@v0.0.0-20210804190922-7d3d774145ac

```

### PR DESCRIPTION
**Pull request by deining**
This PR closes #1281 by adding the neceessary mount to the module configuration.

### CODE REVIEW

**Review by chalin — APPROVED** [review]
Thanks for the fix, @deining. Do you want to wait for feedback from @roidelapluie before this gets merged, or shall I go ahead and merge this now?

**Comment by deining** [comment]
> Thanks for the fix, @deining. Do you want to wait for feedback from @roidelapluie before this gets merged, or shall I go ahead and merge this now?

Please go ahead and merge this now. I'm very confident this PR will fix the issue raised.

**Pull request merged**
